besu/src/main/java/org/hyperledger/besu/cli/options/unstable/EvmOptions.java

```diff
@@ -29,6 +31,8 @@ public class EvmOptions implements CLIOptions<EvmConfiguration> {
   /** The constant JUMPDEST_CACHE_WEIGHT. */
   public static final String JUMPDEST_CACHE_WEIGHT = "--Xevm-jumpdest-cache-weight-kb";
 
+  public static final String KZG_TRUSTED_SETUP_FILE_PATH = "--evm-kzg-trusted-setup-path";
```

Comment:
I think this should be an `--X` option.  Once the ceremony is done we will have the kzg embedded in the distribution, correct?  So this is just transitional for testing?